scripts/inactive_issue_checker_test.py

```diff
@@ -0,0 +1,766 @@
+        success = self.service.unassign_issue(issue)
+
+        self.assertFalse(success)
+        mock_delete.assert_called_once_with(
+            f'{self.base_url}/issues/1/assignees',
+            headers=self.service.rest_headers,
+            json={'assignees': ['user1']},
+            timeout=10
+        )
+
+    @mock.patch('requests.delete')
+    def test_unassign_issue_null_response(
+        self, mock_delete: mock.MagicMock
+    ) -> None:
+        """Test unassigning an issue with null response."""
+        mock_delete.return_value = None
+
+        issue = checker.Issue(1, 'user1', 'events_url')
+        with self.assertRaises(AssertionError) as context:
+            self.service.unassign_issue(issue)
+        self.assertEqual(
+            str(context.exception),
+            'Received null res while unassigning issue'
+        )
+        mock_delete.assert_called_once_with(
+            f'{self.base_url}/issues/1/assignees',
+            headers=self.service.rest_headers,
+            json={'assignees': ['user1']},
+            timeout=10
+        )
+
+    @mock.patch('requests.delete')
+    def test_unassign_issue_exception(
+        self,
+        mock_delete: mock.MagicMock
+    ) -> None:
+        """Test unassigning an issue with exception."""
+        mock_delete.side_effect = Exception('Network error')
+
+        issue = checker.Issue(1, 'user1', 'events_url')
+        with self.assertRaises(Exception) as context:
+            self.service.unassign_issue(issue)
+        self.assertEqual(
+            str(context.exception),
+            'Network error'
+        )
+        mock_delete.assert_called_once_with(
+            f'{self.base_url}/issues/1/assignees',
+            headers=self.service.rest_headers,
+            json={'assignees': ['user1']},
+            timeout=10
+        )
+
+    @mock.patch('requests.post')
+    def test_add_alert_comment_on_issue_none_response(
+        self,
+        mock_post: mock.MagicMock
+    ) -> None:
+        """Test handling of None response when adding alert comment."""
+        mock_post.return_value = None
+        issue = checker.Issue(1, 'user1', 'events_url')
+
+        with self.assertRaises(AssertionError) as context:
+            self.service.add_alert_comment_on_issue(issue)
+
+        self.assertEqual(
+            str(context.exception),
+            'Received null res while commenting on issue'
+        )
+
+    @mock.patch('requests.post')
+    def test_add_alert_comment_on_issue(
+        self,
+        mock_post: mock.MagicMock
+    ) -> None:
+        """Test adding alert comment on an issue."""
+        mock_response = mock.Mock()
+        mock_response.status_code = 201
+        mock_post.return_value = mock_response
+
+        issue = checker.Issue(1, 'user1', 'events_url')
+        self.service.add_alert_comment_on_issue(issue)
+
+        expected_comment = (
+            f'Hi @{issue.assignee_username} this PR is inactive '
```

Comment:
Hardcode expected messages fully, don't interpolate anything. Try not to have logic when computing expected values.